src/runtime/entitlements-manager.js

```diff
@@ -244,26 +252,50 @@ export class EntitlementsManager {
       default:
         return;
     }
-    this.postEntitlementsRequest_(new EntitlementJwt(), result, source);
+    const token = parseQueryString(this.win_.location.search)['gaa_n'];
+    let isUserRegistered = null;
+    if (
+      'additionalParameters' in event &&
+      event.additionalParameters !== null &&
+      event.additionalParameters.getIsUserRegistered() !== null
+    ) {
+      isUserRegistered = event.additionalParameters.getIsUserRegistered();
+    }
```

Comment:
```suggestion
    const isUserRegistered = event?.additionalParameters?.getIsUserRegistered?.();
```

The optional chaining operator can simplify this
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining